osmdroid-android/src/main/java/org/osmdroid/views/overlay/DefaultOverlayManager.java

```diff
@@ -128,14 +128,6 @@ public void onDraw(final Canvas c, final MapView pMapView) {
             }
         }
 
-        if (mTilesOverlay!=null)
```

Comment:
what's up with this delete?

Reply:
Oh, it's just a duplicate code - though it's not obvious just looking at the diff.
It's more obvious in the [current version of `DefaultOverlayManager`](https://github.com/osmdroid/osmdroid/blob/master/osmdroid-android/src/main/java/org/osmdroid/views/overlay/DefaultOverlayManager.java#L120).
I guess it's a github merge byproduct.